.github/workflows/issue-manager.yml

```
@@ -0,0 +1,30 @@
+name: Issue Manager
+
+on:
+  schedule:
+    - cron: "0 0 * * *"
+  issue_comment:
+    types:
+      - created
+  issues:
+    types:
+      - labeled
+  pull_request_target:
+    types:
+      - labeled
+  workflow_dispatch:
+
+jobs:
+  issue-manager:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: tiangolo/issue-manager@0.5.0
+        with:
+          token: ${{ secrets.GITHUB_TOKEN }}
```

Shall we use the bot token? :D

```suggestion
          token: ${{ secrets.BOT_TOKEN }}
```

---

github token is usually the token created specifically for this pipeline in this repository, so I believe it's safer to use that.

---

what makes using another token less safe? 😊

---

> what makes using another token less safe? 😊

I think `GITHUB_TOKEN` will only have permissions for stuff in the current pipeline, while `BOT_TOKEN` has a lot more permissions, which could give too much power to a 3rd party lib (the action itself) that we don't control.

Having said that, this comes from a very trusted source, so I'm not against using `BOT_TOKEN` if we prefer it :)